openslides_backend/action/actions/motion/create.py

```diff
@@ -146,19 +147,25 @@ def check_permissions(self, instance: dict[str, Any]) -> None:
             if not has_perm(self.datastore, self.user_id, perm, instance["meeting_id"]):
                 raise MissingPermission(perm)
 
-        # whitelist the fields depending on the user's permissions
+        # Whitelist the fields depending on the user's permissions. Each field can require multiple conjunctive permissions.
         whitelist = []
-        forbidden_fields = set()
-        perm = Permissions.AgendaItem.CAN_MANAGE
-        if has_perm(self.datastore, self.user_id, perm, instance["meeting_id"]):
-            whitelist = [*agenda_creation_properties.keys()]
-        elif contained := set(agenda_creation_properties.keys()).intersection(instance):
-            forbidden_fields.update(contained)
-        perm = Permissions.Mediafile.CAN_SEE
-        if has_perm(self.datastore, self.user_id, perm, instance["meeting_id"]):
-            whitelist.append("attachment_mediafile_ids")
-        elif "attachment_mediafile_ids" in instance:
-            forbidden_fields.add("attachment_mediafile_ids")
+        forbidden_fields = defaultdict(list)
+        permission_to_fields: dict[Permission, list[str]] = {
+            Permissions.AgendaItem.CAN_MANAGE: list(agenda_creation_properties.keys()),
+            Permissions.Mediafile.CAN_SEE: ["attachment_mediafile_ids"],
+            Permissions.Motion.CAN_MANAGE_METADATA: [
+                "additional_submitter",
+                "submitter_ids",
+            ],
+            Permissions.User.CAN_SEE: ["submitter_ids"],
+        }
+        for perm, fields in permission_to_fields.items():
+            if has_perm(self.datastore, self.user_id, perm, instance["meeting_id"]):
+                whitelist.extend(fields)
```

Comment:
You put submitter_ids on the whitelist, even when not UserCanSee, that is a problem.

Reply:
This is now prevented.